.gitignore

```diff
@@ -0,0 +1,7 @@
+#node modules
+node_modules/
+
+#package config files
+package.json
+package-lock.json
+webpack.config.json
```

Comment:
please, explude package.json and webpack.config.json from .gitignore. It was my mistake I told you package.json is not needed. I was thinking about package-lock.json.

Also keep in mind there always a need of an empty line in the end of the file